### PR TITLE
Update to rails 5.1.

### PR DESCRIPTION
**Pull request by SuperQ**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -2,44 +2,44 @@ GEM
...
+      rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.0.7)
-      activesupport (= 5.0.7)
+    actionview (5.1.6)
+      activesupport (= 5.1.6)
       builder (~> 3.1)
-      erubis (~> 2.7.0)
+      erubi (~> 1.4)
```
Comment (danhimalplanet) [inline Gemfile.lock]: is erubis and erubi the same thing?

Comment (SuperQ) [reply Gemfile.lock]: Hrm, not sure, this is what `bundle update` did for me.

It looks like `erubis` is an old, obsolete, ERB parser. `erubi` looks like the replacement.

Comment (danhimalplanet) [reply Gemfile.lock]: ok LGTM

**Comment by danhimalplanet** [comment]
Looks good to me.

**Pull request closed without merging**